DocsClientYamlTestSuiteIT rollover-index access control exception

**Build scan**:
https://gradle-enterprise.elastic.co/s/jsnbzgebcjkzq
https://gradle-enterprise.elastic.co/s/d2kntwkpocm2s
https://gradle-enterprise.elastic.co/s/623erfrzv3ygu
https://gradle-enterprise.elastic.co/s/ernce2n3aqwyg

**Repro line**:
```
./gradlew ':docs:integTest' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/indices/rollover-index/line_372}" \
  -Dtests.seed=35E5D24116483E13 \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-SD \
  -Dtests.timezone=Asia/Ho_Chi_Minh \
  -Druntime.java=11 \
  -Dtests.fips.enabled=true
```

**Reproduces locally?**:
No

**Applicable branches**:
master, 7.x

**Failure history**:
Many since 2nd September

**Failure excerpt**:
```
java.lang.AssertionError: Failure at [reference/indices/rollover-index:5107]: expected [2xx] status code but api [raw[method=PUT path=logs_write/_doc/1]] returned [500 Internal Server Error] [{"error":{"root_cause":[{"type":"access_control_exception","reason":"access denied (\"java.lang.RuntimePermission\" \"accessDeclaredMembers\")","stack_trace":"[access denied (\"java.lang.RuntimePermission\" \"accessDeclaredMembers\")]; nested: AccessControlException[access denied (\"java.lang.RuntimePermission\" \"accessDeclaredMembers\")]
```

--- a/docs/reference/indices/rollover-index.asciidoc
+++ b/docs/reference/indices/rollover-index.asciidoc
@@ -371,7 +371,7 @@ which is incremented every time the index is rolled over. For instance:
 [source,console]
 --------------------------------------------------
 # PUT /<logs-{now/d}-1> with URI encoding:
-PUT /%3Clogs-%7Bnow%2Fd%7D-1%3E <1>
+PUT /%3Clogs_%7Bnow%2Fd%7D-1%3E <1>
 {
   "aliases": {
     "logs_write": {}
@@ -394,10 +394,10 @@ POST /logs_write/_rollover <2>
   }
 }
 --------------------------------------------------
-// TEST[skip:"AwaitsFix https://github.com/elastic/elasticsearch/issues/62043"]
+// TEST[s/now/2016.10.31%7C%7C/]
 
-<1> Creates an index named with today's date (e.g.) `logs-2016.10.31-1`
-<2> Rolls over to a new index with today's date, e.g. `logs-2016.10.31-000002` if run immediately, or `logs-2016.11.01-000002` if run after 24 hours
+<1> Creates an index named with today's date (e.g.) `logs_2016.10.31-1`
+<2> Rolls over to a new index with today's date, e.g. `logs_2016.10.31-000002` if run immediately, or `logs-2016.11.01-000002` if run after 24 hours
 
 //////////////////////////
 
@@ -410,12 +410,12 @@ GET _alias
 [source,console-result]
 --------------------------------------------------
 {
-  "logs-2016.10.31-000002": {
+  "logs_2016.10.31-000002": {
     "aliases": {
       "logs_write": {}
     }
   },
-  "logs-2016.10.31-1": {
+  "logs_2016.10.31-1": {
     "aliases": {}
   }
 }

